source/likely.js

```diff
@@ -1,6 +1,18 @@
 'use strict';
 
 var likely = require('./index.js');
+var history = require('./history');
+var utils = require('./utils');
 
 window.likely = likely;
-window.addEventListener('load', likely.initiate);
+window.addEventListener('load', function () {
+    if (likely.defaults.listenChangesUrl) {
```

Comment:
🐔

What are the real cases when listening for URL changes should be disabled? I can’t come up with any. Probably you have some in mind?

(Edited your first comment to also include 🐔)